lib/errors.ts

```diff
@@ -99,6 +100,7 @@ export const GitErrorRegexes = {
   "fatal: .+: '(.+)' is outside repository": GitError.OutsideRepository,
   'Another git process seems to be running in this repository, e.g.':
     GitError.LockFileAlreadyExists,
+  'fatal: There is no merge to abort': GitError.NoMergeToAbort,
```

Comment:
@shiftkey does the somewhat looser regex look ok to you? i didn't exhaustively look into different merge conflict failings so i thought that perhaps casting a slightly wider net would be ok?

Reply:
We can always come back and refine the regex later while we iterate on the feature :shipit: